src/arche/quality_estimation_algorithm.py

```diff
@@ -126,8 +101,7 @@ def generate_quality_estimation(
     else:
         quality_estimation = (
             adherence_to_schema_percent * 40 / 100
-            + duplicated_items_percent * 10 / 100
-            + duplicated_skus_percent * 5 / 100
+            + duplicated_items_percent * 15 / 100
```

Comment:
From where these percentages come from? Why 15 percent of duplicated_items_percent?

Reply:
They were set 1.5 years ago based on the experience of qa team at that moment.
I set 15 because it's the sum for duplicates, to keep the compatibility for now, see #154